test/features.jl

```diff
@@ -350,10 +350,10 @@ end
 
 @test Zygote.@code_adjoint(f(1)) isa Zygote.Adjoint
 
-@test_throws ErrorException Zygote.gradient(1) do x
-  push!([], x)
-  return x
-end
+# @test_throws ErrorException Zygote.gradient(1) do x
+#   push!([], x)
+#   return x
+# end
```

Comment:
```suggestion
@test_throws ErrorException Zygote.gradient(1) do x
  push!([], x)
  return x
end
```
This should work again, if I'm not mistaken?

Reply:
Yes I think so, I missed that.